javalib/src/main/scala/java/util/concurrent/Throwables.scala

```diff
@@ -17,3 +17,13 @@ class CancellationException(message: String)
 class TimeoutException(message: String) extends Exception(message) {
   def this() = this(null)
 }
+
+class RejectedExecutionException(message: String, cause: Throwable)
+    extends RuntimeException(message) {
+  
+  def this() = this(null, null)
+
+  def this(message: String) = this(message, null)
+
+  def this(cause: Throwable) = this(null, cause)
```

Comment:
The doc says that the message should be ` (cause == null ? null : cause.toString())`